core/CMakeLists.txt

```diff
@@ -126,10 +126,17 @@ target_include_directories(tangram-core
 
 # Add JavaScript implementation.
 if(TANGRAM_JSCORE_ENABLED)
+
+  target_compile_definitions(tangram-core
+    PRIVATE
```

Comment:
Using a private compile definition here means that the benchmark targets will never get `TANGRAM_USE_JSCORE=1`

Reply:
Yeah so I've added it to benchmarks. It's not really of public interest though :)

Reply:
I was thinking of making this private for a libtangram-stylecontext to have less to rebuild when switching js implementations.

Reply:
Agree that a public flag isn't ideal. 

I can see the logic in making a stylecontext library, but I think switching JS engines is going to be pretty infrequent in practice.